Python/ceval.c

```diff
@@ -3174,9 +3174,10 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, PyFrameObject *f, int throwflag)
                     la = &co_opcache->u.la;
                     if (la->type == type && la->tp_version_tag == type->tp_version_tag)
                     {
-                        // Hint >= 0 is a dict index; hint < 0 is an inverted slot index.
-                        if (la->hint < 0) {
-                            /* Even faster path -- slot hint */
+                        // Hint >= 0 is a dict index; hint == -1 is a dict miss.
+                        // Hint < -1 is an inverted slot offset (offsets are never 0).
```

Comment:
Maybe clarify that slot offsets are always greater than 1?

Reply:
I have a comment ready to push:
```
                        // Hint >= 0 is a dict index; hint == -1 is a dict miss.
                        // Hint < -1 is an inverted slot offset: offset is strictly > 0,
                        // so ~offset is strictly < -1 (assuming 2's complement).
```
Let me know if you still want improvement.

Reply:
looks good to me